src/global intents.json

```diff
@@ -1,36 +1,36 @@
 {
     "intents": [
         {
-            "name": "StartCall",
+            "name": "fdc3.startCall",
             "displayName": "Call" 
         },
         {
-            "name": "StartChat",
+            "name": "fdc3.startChat",
             "displayName": "Chat"
         },
         {
-            "name": "ViewChart",
+            "name": "fdc3.viewChart",
             "displayName": "Chart"
         },
         {
-            "name": "ViewContact",
+            "name": "fdc3.viewContact",
             "displayName": "View"
         },
         {
-            "name": "ViewQuote",
+            "name": "fdc3.viewQuote",
             "displayName": "Quote"
         },
         {
-            "name": "ViewNews",
+            "name": "fdc3.viewNews",
             "displayName": "News"
         },
         {
-            "name": "ViewInstrument",
+            "name": "fdc3.viewInstrument",
             "displayName": "Details"
         },
         {
-            "name": "ViewAnalysis",
+            "name": "fdc3.viewAnalysis",
             "displayName": "Analyze"
         }
     ]
-}
\ No newline at end of file
+}
```

Comment:
So I am not sure we should add an FDC3 root namespace to intent names. We are using the vanilla StartCall intent internally, and it is quite clear to consumers because it has no namespace that it is "global".

I do think context types should always have a namespace, because I am not sure there is a "global" type/domain model the same way as for intents. E.g. it is always FDC3, or FINOS, or Symphony, or a particular organisation's view on what a "contact" is.